StudySharp.Domain/Models/Learning/Content.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections.Generic;
+
+namespace StudySharp.Domain.Models.Learning
+{
+    public class Content
+    {
+        public int Id { get; set; }
+        public Course Course { get; set; }
```

Comment:
How do you get a Course model without CourseId?

@KolaCation we really need Id here?

Reply:
For the 1 to 1 connection we need it